Gemfile

```diff
@@ -3,7 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in tsv.gemspec
 gemspec
 
-gem "codeclimate-test-reporter", group: :test, require: nil
-gem "rake"
-gem "rspec"
-gem "pry"
+group "test" do
```

Comment:
I had a pause remembering how it's used. Explicit grouping doesn't change anything functionally, but adds visual reminder.